erpnext/manufacturing/doctype/work_order/work_order.js

```diff
@@ -442,6 +442,18 @@ frappe.ui.form.on("Work Order", {
 	additional_operating_cost: function(frm) {
 		erpnext.work_order.calculate_cost(frm.doc);
 		erpnext.work_order.calculate_total_cost(frm);
+	},
+
+	onload_post_render: function(frm) {
+		if (frm.doc.transfer_material_against == "Job Card") {
+			frappe.msgprint(__("Please add an operation in required items table"));
+		}
+	},
+
+	transfer_material_against: function(frm) {
+		if (frm.doc.transfer_material_against == "Job Card") {
+			frappe.msgprint(__("Please add an operation in required items table"));
+		}
```

Comment:
Why is this message required? Also, why is it always shown regardless of the content of the operations table? 

Maybe the user has added operations? Maybe operations will get fetched from BOM?